spec/features/badge_spec.rb

```diff
@@ -61,20 +49,58 @@ def log_user(login, password)
       check 'pref_issue_badge'
       click_on 'Save'
 
-      expect(page).to have_selector("#issue_badge")
+      expect(page).to have_selector('#issue_badge')
+    end
 
-      user = User.where(:login => 'dlopper').first
-      all_issues = Issue.visible.open.where(:assigned_to_id => ([user.id] + user.group_ids))
-      puts "dlopper's issue: #{all_issues.length}"
-      if all_issues.any?
-        expect(page).to have_selector("#issue_badge_number")
-        expect(page).to have_selector("#issue_badge_number"), text: all_issues.length
+    scenario 'Badge number is displayed if badge option is activated and operator has assigned issues.' do
```

Comment:
__[rubocop]__
* Line is too long. [106/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)